docs/providers/aws/guide/serverless.yml.md

```diff
@@ -928,6 +928,32 @@ functions:
           enabled: false
 ```
 
+### ActiveMQ
+
+[ActiveMQ events](../events/activemq.md):
+
+```yaml
+functions:
+  hello:
+    # ...
+    events:
+      - activemq:
+          # ARN of ActiveMQ Broker
+          arn: arn:aws:mq:us-east-1:0000:broker:ExampleMQBroker:b-xxx-xxx
+          # name of ActiveMQ queue consume from
+          queue: queue-name
+          # Secrets Manager ARN for credentials required to do basic auth
+          basicAuthArn: arn:aws:secretsmanager:us-east-1:01234567890:secret:MySecret
+          # Optional, must be in 1-10000 range
```

Comment:
Let's be consistent with specifying ranges - below you're stating what is the max and min values and here you reference it as a range - is there a reason for that? I would stick to stating what is min and what is max value

Reply:
Fixed in 952ae3e404ad26c6331dd3f882ea7137efaeb031

Reply:
I'd copied it from the MSK example above, so will amend that too